Updates for PM review. (#35)
mulesoft/docs-partner-manager: modules/ROOT/pages/B2B-overview.adoc

> Anypoint Partner Manager allows you to manage a large number of trading partners and their message flows. A message flow is the path along which a message travels, from its source, through a transformation (via a map), and to its target. You'll need to perform some tasks before you can begin transmitting messages with your trading partners. A transmission is a single communication event where a transfer of a payload which may contain one or more messages. In the X12 standard, messages are the same as transaction sets.

Reviewer: allows > enables
This is per our style reference. Do not let your reviewers revert.

Reviewer: Suggest breaking para at "You'll need to...."

Reviewer: where > during which

Reviewer: "A transmission is a single communication event where a transfer of a payload which may contain one or more messages."  >
"A transmission is a single communication event during which payload transfer sometimes requires more than one message."

The original sentence is not grammatical. Do not let reviewers revert.

Author: Fixed.

> You set yourself up as a host partner, then configure your partners, and then create message flows between you and your partners.

Reviewer: This sentence should logically follow the sentence about needing to perform some tasks...those tasks being "First, you set yourself as host partner. Then you configure your partners, and finally create message flows between you and your partners."

Suggest following with "Even though a transmission is a single communication event, it sometimes requires more than one message (or transaction set) to transfer payload."

The original sentences sound choppy, without a logical flow or transitions between them. Unless this affects technical meaning, I suggest that you not revert.

> 

Reviewer: This introduction is not consistent with the text that follows it. It mentions info from partners and then understanding your backend, but then it describes these things in the reverse order. The two should match..

Reviewer: The final sentence above is unnecessary. What follows is not a checklist; it describes what the previous two sentences introduced. Suggest omitting, especially since the sentence begins with "Use this..." and does not end with a colon pointing to anything.

> Information you'll need to have on hand as the host organization:

Reviewer: "on hand" is colloquial and difficult to translate.
If this is the aforementioned "understanding of how your backend systems work," I suggest rewording either that or this so that they match.
It seems more logical to change the introduction to state simply that you need certain information about your system as host, and your partners' systems: 

Information you need about your organization:

> * Your organization's identifiers (such as the AS2 Identifier, X12 ISA, X12 GS, and so on)

Reviewer: AS2 Identifier > AS2 ID

I looked this up and saw other versions as well: AS2 System Identifier, for one. What was standard was use of AS2 ID, which is parallel with the other items in this list.

> * Endpoints: There are three types of endpoints within AnyPoint Partner Manager -

Reviewer: QUESTION: This list is about host info, but it looks like one of these endpoints is actually partner info (acknowledgement endpoint)?
If so, suggest changing this bullet item to...

   * Your receive and target endpoints:
      **Receive endpoints
         Exposed to your trading partners and used by your organization to receive transmissions from them
     **Target endpoints
        Host-internal endpoints to which you route messages from your trading partners, after those messages
        are transformed in the message flow

Author: I'm gonna need to check with the PM on the usage here.

> ** Receive endpoints - On the Host side, for an inbound transmission you need endpoints your organization uses to receive transmissions from your partners. These endpoints are resources your organization exposes to your trading partners.

Reviewer: See previous suggestion.
If you keep this text, these copyedits:

```suggestion
** Receive endpoints
    As host, endpoints your organization uses to receive transmissions from your partners. These endpoints are resources your organization exposes to your trading partners.
```

Author: Again, from previous.

> ** Target endpoints - As the Host, your organization has internal endpoints where you route messages from the Partner, after they are transformed within the message flow.

Reviewer: See previous suggestion.
If you keep this text, these copyedits:

```suggestion
** Target endpoints
    As host, internal endpoints to which you route messages from the partner, after those messages are transformed within the message flow.
```

> ** Acknowledgement endpoints - Your partners provide you with endpoint configurations that can be used to send acknowledgement messages like 997s or 999s back to them. Acknowledgements are typically sent back to trading partners corresponding to one of more B2B messages you may receive from them.

Reviewer: This belongs in the second list, of information that the host needs to obtain from the partner.
Suggest moving, with these copyedits:

```suggestion
* Partner acknowledgment endpoints
  Your partners provide you with endpoint configurations that you use as targets for sending messages (like 997s or 999s) to trading partners, acknowledging receipt of their transmissions.
```

Author: I'm going to have to review this with the PM. If it sounds confusing, it's because as the host, you do some partner configurations you need to communicate before you configure a partner. They kind of reuse the UI in a way that makes it a bit complicated.

* Any Dataweave transformation maps your organization uses for B2B

> Information needed for each trading partner:

Reviewer: ```suggestion
Information you need about each trading partner:
```

> * Partner name (usually the name of the company)

Reviewer: For text below, which I cannot alter:

   * Partner identifiers (such as AS2 ID, ....)
   * Partner acknowledgment endpoints
     [[see above]]
   * Electronic...
   * Message types to use
      For example....
   * Partner digital certificates....
